### DISCUSSION TITLE
links compatible

### DISCUSSION THREAD

**Discussion by manas-jaiswal** [deepgram/community#365] [General help] [answered]
Wether a livestream m3u8 link is compatible for deepgram models????

**Comment by shirgoldbird** [comment, marked as answer]
Hi @manas-jaiswal! We don't support m3u8 livestreams and don't plan to add support for them at this time. If you'd like to send m3u8 audio to Deepgram, you'll first need to fetch the audio from the livestream and then convert it to one of our [supported audio formats](https://developers.deepgram.com/docs/supported-audio-formats).

Here's an example of how to convert m3u8 audio to an mp4 file, which can then be sent to Deepgram for processing:

`fmpeg -y -f hls -i YOUR_FILE.m3u8 -bsf:v h264_mp4toannexb,dump_extra -bsf:a aac_adtstoasc -map 0 -c copy -flags +global_header -max_interleave_delta 0 -f mp4 test.mp4`
